blockchain/chain_sync/src/sync.rs

```diff
@@ -136,306 +128,226 @@ where
     }
 
     /// Returns the atomic reference to the syncing state.
-    pub fn sync_state_cloned(&self) -> Arc<RwLock<SyncState>> {
-        self.state.clone()
+    pub fn sync_state_cloned(&self) -> WorkerState {
```

Comment:
Fix the commend for this funcc

Reply:
I mean, it is still correct? The type is just aliased because it's verbose

Reply:
Hm, yeah youre right. I got a little confused. This is the nested arc and rwlock madness. 
Nit: but i would change it to specify its the worker sync state and not the overall chainsync sync state. Not a real big issue, ignore if you want